requirements.txt

```diff
@@ -28,3 +28,5 @@ langid==1.1.6
 tqdm
 tensorboard==2.16.2
 loguru==0.7.2
+botocore>=1.34.98
+cached_path>=1.6.2
```

Comment:
At the top there is a `cached_path` as well, I think we can remove that one.

Reply:
Done. Please review again.